styles/main.css

```diff
@@ -8,25 +8,25 @@ body {
 h1 {
     color: #6e309e; 
     background-repeat: repeat-x; 
-    font-family: 'Proza Libre', sans-serif;
+    font-family: 'Proza Libre', sans-serif;
     text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.2);
     background: rgb(2,0,36);
     background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(190,0,222,1) 0%, rgba(189,33,203,1) 23%, rgba(207,183,212,1) 100%);
     background-clip: text;
     -webkit-background-clip: text;
-    color: transparent;
+    color: transparent;
 }
 
 h2 {
     color: #6e309e; 
     background-repeat: repeat-x; 
-    font-family: 'Proza Libre', sans-serif;
+    font-family: 'Proza Libre', sans-serif;
     text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.2);
     background: rgb(2,0,36);
     background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(190,0,222,1) 0%, rgba(189,33,203,1) 23%, rgba(207,183,212,1) 100%);
     background-clip: text;
     -webkit-background-clip: text;
-    color: transparent
+    color: transparent

```

Comment:
Not sure why GitHub is showing these as changed... I did not change these lines, and they look the same to me here...